test/__mocks__/date-fns.js

```diff
@@ -0,0 +1,3 @@
+export const formatDistance = (date, baseDate) => `${+date} [${baseDate}]`;
+
+export const subSeconds = (date, amount) => `${+date} [${amount}]`;
```

Comment:
`date-fns` should be in `devDependencies`, it's currently missing 😉 

You can add it with `yarn add date-fns --dev`

Reply:
oh, I did it several times

Reply:
![image](https://user-images.githubusercontent.com/27810942/67563518-2946d480-f721-11e9-9536-64ed4f273eeb.png)

still missing...

Reply:
Maybe you forgot to commit ;)